blueprints/secops/bindplane-gke/README.md

```diff
@@ -0,0 +1,155 @@
+| [prefix](variables.tf#L80) | Prefix used for resource names. | <code>string</code> | ✓ |  |
+| [project_id](variables.tf#L99) | Project id, references existing project if `project_create` is null. | <code>string</code> | ✓ |  |
+| [region](variables.tf#L104) | GCP region. | <code>string</code> | ✓ |  |
+| [bindplane_config](variables.tf#L17) | Bindplane config. | <code title="object&#40;&#123;&#10;  tls_certificate_cer &#61; optional&#40;string, null&#41;&#10;  tls_certificate_key &#61; optional&#40;string, null&#41;&#10;&#125;&#41;">object&#40;&#123;&#8230;&#125;&#41;</code> |  | <code>&#123;&#125;</code> |
+| [cluster_config](variables.tf#L37) | GKE cluster configuration. | <code title="object&#40;&#123;&#10;  cluster_name &#61; optional&#40;string, &#34;bindplane-op&#34;&#41;&#10;  master_authorized_ranges &#61; optional&#40;map&#40;string&#41;, &#123;&#10;    rfc-1918-10-8 &#61; &#34;10.0.0.0&#47;8&#34;&#10;  &#125;&#41;&#10;&#125;&#41;">object&#40;&#123;&#8230;&#125;&#41;</code> |  | <code>&#123;&#125;</code> |
+| [dns_config](variables.tf#L48) | DNS config. | <code title="object&#40;&#123;&#10;  bootstrap_private_zone &#61; optional&#40;bool, false&#41;&#10;  domain                 &#61; optional&#40;string, &#34;example.com&#34;&#41;&#10;  hostname               &#61; optional&#40;string, &#34;bindplane&#34;&#41;&#10;&#125;&#41;">object&#40;&#123;&#8230;&#125;&#41;</code> |  | <code>&#123;&#125;</code> |
+| [postgresql_config](variables.tf#L70) | Cloud SQL postgresql config. | <code title="object&#40;&#123;&#10;  availability_type &#61; optional&#40;string, &#34;REGIONAL&#34;&#41;&#10;  database_version  &#61; optional&#40;string, &#34;POSTGRES_13&#34;&#41;&#10;  tier              &#61; optional&#40;string, &#34;db-g1-small&#34;&#41;&#10;&#125;&#41;">object&#40;&#123;&#8230;&#125;&#41;</code> |  | <code>&#123;&#125;</code> |
+| [project_create](variables.tf#L90) | Provide values if project creation is needed, uses existing project if null. Parent is in 'folders/nnn' or 'organizations/nnn' format. | <code title="object&#40;&#123;&#10;  billing_account_id &#61; string&#10;  parent             &#61; string&#10;&#125;&#41;">object&#40;&#123;&#8230;&#125;&#41;</code> |  | <code>null</code> |
+
+## Outputs
+
+| name | description | sensitive |
+|---|---|:---:|
+| [bindplane_hostname](outputs.tf#L17) | BindPlane OP Management console hostname. |  |
+| [ca_cert](outputs.tf#L22) | TLS CA certificate. |  |
+| [cluster_ca_certificate](outputs.tf#L27) | GKE CA Certificate. |  |
+| [fleet_host](outputs.tf#L32) | GKE Fleet host. |  |
+| [lb_ip_address](outputs.tf#L37) | Ingress LB address. |  |
+<!-- END TFDOC -->
+## Test
+
+```hcl
+module "bindplane-gke" {
+  source     = "./fabric/blueprints/secops/bindplane-gke"
+  project_id = "test"
+  bindplane_secrets = {
+    user            = "admin"
+    password        = "thisisnotasecret"
+    secret_key      = "xxxxxx-xxxxxxx-xxxxxx"
```

Comment:
This readme needs a warning stating that the state will contain these secrets.

Reply:
Added warning section in the README file